MetaMask/metamask-extension
# feat: setup feature flag for the new add network ui

## **Description**
The goal of this PR is to add a new environment variable allowing us to manage the feature flag for the new add/edit network UI flow


[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/24903?quickstart=1)

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to github action jobs
2. Find the test-e2e-chrome-network-ui-redesign or test-e2e-firefox-network-ui-redesign
3. Confirm it displays

## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
